app/payments/get-payments-by-scheme-id.js

```diff
@@ -0,0 +1,11 @@
+const { SCHEME_ID } = require('../constants/categories')
+const { SCHEME_ID_VALUE } = require('../constants/scheme-id-value')
+const { getData } = require('./get-data')
+
+const getPaymentsBySchemeId = async (schemeId = SCHEME_ID_VALUE) => {
```

Comment:
`getData` function requires a value property but not required anywhere else to get the events by scheme as filtering by category. 
Therefore used a default param here but unsure if there is a better way so open to suggestions here.

Reply:
This could be adapted for future tickets to only return events for specific scheme however as the default view is to display all scheme data its unlikly this will be required and filtering to specific scheme will be done on frontend in my opinion.

Reply:
So the schemeIdValue will always be 1? I'm not too sure I understand what this is doing

Reply:
The Data Hub requires both a `category` and a `value` for every request it gets, otherwise it rejects the message.

So the options are:
1. Update the Data Hub's validation and just pass the `value` as `undefined`.
2. Pass a default value to pass the validation (what you are doing).

I feel that 2 is probably the corect approach, but I think the value being set to '1' is misleading as it wrongly implies other numbers can be passed to get different results.  

I'd suggest changing the value to `all` or `default` or something that indicates every scheme will be returned.

Also not sure if we should refer to the function, variable and constant as `schemeId` as we're not actually getting or filtering data based on schemeId.  Would `getPaymentsByScheme`, `SCHEME_VALUE`, `scheme` be better?

Reply:
gone with option 2 and renamed references